massdriver-application-gcp-cloud-function/artifacts.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  data_infrastructure = {
```

Comment:
work in progress with what we had from functions as infra bundles. Ready to match it with app artifacts when that is ready.